docs/local_install_docker_compose.md

```diff
@@ -106,36 +143,33 @@ redis-server /usr/local/etc/redis.conf
 
 - Install dependencies and start the frontend:
 
-```bash
+```sh
 cd $SUBSTRA_SOURCE/substra-frontend
 yarn install
 yarn start
 ```
 
-- Go to http://localhost:3000/ in your browser to access the web interface.
-
-## Add some assets to Substra
-
-- Install populate script dependencies:
+You can now go to <http://localhost:3000/> in your browser to access the web interface!
 
-```bash
-pip install termcolor
-```
+## 6. Add some assets to Substra
```

Comment:
Would be nice to add something like: Now you can use it :-) And link to the examples. Maybe only mention that the `populate` is just a way to test if it works, and not the classic way to use it